test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -174,8 +195,20 @@ function RunUpdateHelpTests
     {
         if ($powershellCoreModules -contains $moduleName)
         {
+            if ($moduleName -eq "Microsoft.PowerShell.Core")
+            {
+                $skip = $MicrosoftPowerShellCorePathIsReadOnly
+            }
+            else
+            {
+                $modulePath = (Get-Module -Name $moduleName -ListAvailable).ModuleBase
+                $skip = PathIsReadOnly $modulePath
+            }
+
+            # -Skip and -$Pending are mutually exclusive on It block, so skip Pending tests
```

Comment:
Skip and Pending have specific meanings. you can use splatting to choose one of them for the It block.